### PR TITLE
Added Books in Python

### PR DESCRIPTION
**Pull request by harshdattani**

### CODE REVIEW

**Comment by rathodchintan** [comment]
Thank you @harshdattani  for your contribution. Only thing to mention is formatting. :)

**Comment by harshdattani** [comment]
Yes!

**Comment by harshdattani** [comment]
Formatting done :)

**Comment by rathodchintan** [comment]
Superb.. :+1:

**Comment by PSanni** [comment]
@harshdattani Messing the thing, for commit contains description and long title, please follow this format, as suggested by @PareshMayani  to me. 
https://github.com/GDGAhmedabad/Learning-Resources/blob/master/Python/courses.md

File: Python/books.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Learn Python the Hard Way | http://learnpythonthehardway.org/
```
Comment (PareshMayani) [inline Python/books.md]: Please follow list style for the formatting, check this for reference https://github.com/GDGAhmedabad/Learning-Resources/blob/master/android/Useful_links.md

**Comment by PareshMayani** [comment]
Sorry guys for the confusion about formatting. Let's make sure all the future commits will have "list" formatting followed!

**Pull request merged**